### ISSUE TITLE
Pinned messages are inaccessible

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2050]
When tabbing through a chat, I get the following when there is a pinned message.
* A link with no text (if I use object navigation I can see what it is, this also applies to replies by the way)
* A button to remove the pin, with no label. 

This is quite counterintuitive, as people often don't use object navigation anywhere (I probably discovered it was useful by accident). Also, I cannot reliably jump to the pinned message. Currently the workaround I use is to press enter on the link, go to the message list and just move around near it, as there are also focus issues (think go to end, for example, or the one referring to going to first unread message).

Thanks!

**Comment by sukiletxe** [comment]
This has been fixed in latest beta. The link with no text still shows, but there is now a "Show all pinned messages" button, which opens a list with all the pinned messages (yes, with just one it works too). The only issue is that you cannot reply to the pinned message from there.

**Comment by MarcAnt01** [comment]
@sukiletxe that's not an accessibility issue, there's no button there in Unigram, like in the official clients

**Comment by sukiletxe** [comment]
Seems either that it was not entirely fixed, or that there has been a regression. The problems in the first message which _I reported as fixed (with the see pins list button) aren't present in supergroups with one pinned message (sadly I don't have supergroups with more than one, or groups with only one to test which of them it is which makes the button disappear, though I suspect the quantity)

**Comment by MarcAnt01** [comment]
Can you open a ticket limited to pinned messages of groups with only one lined message?

**Comment by sukiletxe** [comment]
This got fixed by the replies fix. I can confirm that, although the control type (link) is no longer said both in replies and pinned messages, pressing enter takes you to the vicinity of the message in question. The focus issue (#1557) still remains, but the dev has already said he doesn't know how to fix it (see comments in that issue. So, closing.

**Comment by sukiletxe** [comment]
Oh, it was already closed. Anyway, little issue I've found while testing, the pinned message always starts with "#, #" for some reason.
